core/tests/webdriverio_utils/StoryEditorPage.js

```diff
@@ -473,7 +473,8 @@ var StoryEditorPage = function() {
       await nodeOutlineEditorRteContentSelector());
     var outlineEditorRteContentText = (
       await nodeOutlineEditorRteContent[0].getText());
-    expect(outlineEditorRteContentText).toEqual(nodeOutline);
+    var nodeOutlineText = await nodeOutline.replace(/<.+?>/g, '');
```

Comment:
Considering `nodeOutline` is a function param, why can't we pass the required value and test it? Instead of changing it here before assert.

Note: This method will succesfully pass the test for both `'<p>First outline</p>'`, `'<random>First outline</random>'`.

Reply:
Thank you for the review, @DubeySandeep. I'm working on it.